source/docs/appendices/migration-guides.md

```diff
@@ -7,6 +7,140 @@ This area will contain the Migration steps to follow for upgrading your store to
+  required
++ disableShowPassword={true}
+/>
+```
+
+As such we needed a new eye-off icon. We added this icon in [the theme's `<Icon>` component](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/atoms/Icon/Icon.js). If you have overridden the `<Icon>` component please add an icon named `eye-off` to the list of icons.
+
+```diff
+import {
+  ...
++ IoIosEyeOff,
+} from "react-icons/io";
+
+const keyToComponent = {
+  ...
+  eye: IoIosEye,
++ "eye-off": IoIosEyeOff,
+  pencil: IoMdCreate,
+  ...
+};
+```
+
+This new `Password` component requires a stylesheet to add in the `_components.scss` file if you overrode it.
+
+```diff
+...
+@import "~theme/components/atoms/Form/Label/Label";
++@import "~theme/components/atoms/Form/Input/Password/Password";
+@import "~theme/components/atoms/Form/Input/Select/Select";
+...
+```
+
+If you overrode the `_Input.scss` file, you may need to indicate the inputs height by adding an `input-height` class to correct the vertical alignment of the icon. 
+
+```diff
+input,
++.input-height,
+select {
+  height: 3.4rem;
+}
+```
+
+### New PasswordStrengthHint
+
+We included `<PasswordStrengthHint>` to detail the expected complexity of the password.
+
+If you overrode the following files, you will ned to add the two new components introduced by the feature in them :
+
+`_components.scss`
+
+```diff
++@import "~theme/components/atoms/Form/Input/PasswordStrengthHint/PasswordStrengthHint";
++@import "~theme/components/atoms/ProgressStatus/ProgressStatus";
+```
+
+`src/web/theme/components/atoms/Form/Input/index.js`
+
+```diff
++import PasswordStrengthHint from "./PasswordStrengthHint";
+
+export {
+  ...
++ PasswordStrengthHint,
+};
+```
+
+You can deactivate of the `PasswordStrengthHint` feature by adding the variable `FRONT_COMMERCE_WEB_PASSWORD_HINT_DISABLE=true` to your environment 
+
+If you overrode some forms with password creation/change (`RegisterForm`, `ChangeUserPasswordForm`, `PasswordReset`, `PasswordResetForm`), you can add the component this way
+
+```diff
++import PasswordStrengthHint from "theme/components/atoms/Form/Input/PasswordStrengthHint/PasswordStrengthHint";
+
+<Password 
+  name="password"
+  ...
+/>
++ <PasswordStrengthHint
++  formValuePath="password" // must equal the password name value 
++/>
+```
+
+### `passwordValidation` deprecation 
+
+The file `src/web/theme/components/atoms/Form/Input/Password/passwordValidation.js` is now deprecated, if you overrode it, you will ned  to override the password validity configuration in `src/web/theme/components/atoms/Form/Input/Password/passwordConfig.js`.
+
+See ??? for more details on the password field validation configuration.
+
+If you overrode some of the impacted components (`RegisterForm`, `ChangeUserPasswordForm`, `PasswordReset`, `PasswordResetForm`), use the new `src/web/theme/components/atoms/Form/Input/Password/passwordFieldValidator.js` instead of the deprecated `passwordValidation` as follow
```

Comment:
**suggestion:** reuse the same list as above. 

**thought:** I wonder if it wouldn't make more sense to group those sections so that integrators could batch these updates for each file. What do you thing?

Reply:
Those paragraphs are not talking about the same action, the first one is about re-adding the hinter, the second one is about using the new password configuration. 

In my opinion they are two totally different kind of actions.